woocommerce/woocommerce
# Issue/31678

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #31678  .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create an order.
2. Open the order and go to the "Downloadable product permissions" section at the bottom.
3. Make sure you have not searched any product.
4. Click the "Grant access" button.
5. Error message should show.
6. Users should be notified that they need to enter a valid downloadable product so that they can grant access.
7. The "Grant access" button is disable is no product is entered.

![スクリーンショット 2023-11-25 3 40 50](https://github.com/woocommerce/woocommerce/assets/2771396/8da69323-d559-4f27-baad-cb430477bf82)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
